### ISSUE TITLE
Failed to save Plastic Level after applying Autorenumber option to frames in plastic column.

### ISSUE THREAD

**Issue by beeheemooth** [opentoonz/opentoonz#4552]
### Description

I managed to reproduce one of the annoying glitches that cause frames to disappear in Plastic Levels.

**A Plastic Mesh Level becomes unsaveable if its first frame is moved and autorenumbered.**

_Note: I wrote a similar report first in Tahoma2D, but it turned out that it is present in Opentoonz_

### Steps to Reproduce

1. Create a new level (the type doesn't matter) and draw some shape. (You can save the scene, by the way)
2. Apply the Plastic Tool to the drawn shape, as a result a new Plastic Level will be formed.
3. Move the first frame in the plastic mesh column down (or up if you didn't create the level in the first frame)
4. Apply Autorenumber to the moved plastic mesh frame. Note that in the Level Strip, the frame has also changed its number (as it should), and an asterisk has appeared next to the level name, which indicates that the level has changed.
5. Try to apply "Save All" or "Save Level" and you will get a warning window about an error in saving the level. If you ignore it and close the scene, then after restarting the scene, the mesh frame will be red.
If you return the frame to its original position and apply Autorenumber again, the level can be saved.

### OpenToonz Version

Nightly (Specify below)

### OpenToonz Version Information

built Aug 25 2022 17;37;33

### Operating System

Windows

### GPU

NVIDIA

### Graphics Tablet

Huion

**Comment by beeheemooth** [comment]
@RodneyBaker Here I mean a closed and filled shape so that the Plastic Tool can be applied. It doesn't matter how you draw the shape.
I tried to reproduce the problem on another computer and I succeeded.
If I can reproduce the problem in both Tahoma and Opentoonz on different computers, then it is logical to assume that the problem still exists. The fact that you could not reproduce it is probably due to different versions of Windows.

**Comment by RodneyBaker** [comment]
Yes, I agree.  The diffferent versions of Windows is the most likely candidate.

I do have a little suspicion that something like the older logitec webcam that was causing problem with Opentoonz starting... drivers like that... can produce odd things.  

I do also note that you have a Huion tablet which is mentioned often in reports with bugs/crashes.
It would be odd indeed if your reported problem relates to the Huion tablet though.

If someone else on WIndows 8.1 doesn't have the issue then... I'd look for a driver issue.

**Comment by beeheemooth** [comment]
Hmm... Rodney, maybe this is not a tablet or Windows, but a glitch in which the mesh level becomes unsaved after the first frame is renumbered?
 Can you record a video for me, Thomas the Unbeliever, on how it works for you?

**Comment by RattusaurusRex** [comment]
Following these steps results in mesh being unable to save:

1. Create mesh
2. Move frame
3. Save All
4. Autorenumber
5. Save All  

If instead, I skip step 3, the mesh is able to be saved.

https://user-images.githubusercontent.com/60277069/192646869-0e71a559-3a5e-4a6d-ac0c-33444b446205.mp4

**Comment by RodneyBaker** [comment]
Possibly related to [1498](https://github.com/opentoonz/opentoonz/pull/1498) that fixed a similar problem with image levels.  It seems reasonable to me that mesh might be having the same issue with autorenumber.
